models/comment.models.js

```diff
@@ -17,4 +17,10 @@ module.exports.insertComment = (username, body, id) =>{
     .then((result)=>{
         return result.rows[0];
     })
+}
+
+module.exports.removeCommentById = (id)=>{
+    return db.query(
+        `DELETE FROM comments WHERE comment_id = $1`, [id]
```

Comment:
If you use the SQL syntax `RETURNING *;` on the end of this query, then you can do the error checking all inside of this one model 👍